xAPI-Data.md

```diff
@@ -1608,70 +1608,17 @@ The asserting authority represents the authenticating user or some system or app
 
 ###### <a name="2.4.9.s3"></a>Requirements
 
-* <a name="2.4.9.s3.b1"></a>Authority MUST be an Agent, except in 3-legged OAuth, where it MUST be a Group with two Agents. 
-The two Agents represent an application and user together.
-* <a name="2.4.9.s3.b2"></a>The LRS MUST include the user as an Agent as the entire authority if a user connects 
-directly (using HTTP Basic Authentication) or is included as part of a Group. 
-* <a name="2.4.9.s3.b3"></a>The LRS MUST ensure that all Statements stored have an authority.
-* <a name="2.4.9.s3.b4"></a>The LRS SHOULD overwrite the authority on all Statements it stores,
+* <a name="2.4.9.s3.b1"></a>The LRS MUST ensure that all Statements stored have an authority.
+* <a name="2.4.9.s3.b2"></a>The LRS SHOULD overwrite the authority on all Statements it stores,
 based on the credentials used to send those Statements.
+* <a name="2.4.9.s3.b2"></a>When the LRS overwrites the authority, the LRS MUST apply a deterministic process to map
+the credentials used to store a statement to an authority.
 * <a name="2.4.9.s3.b5"></a>The LRS MAY leave the submitted authority unchanged but SHOULD do so only 
 where a strong trust relationship has been established, and with extreme caution.
-* <a name="2.4.9.s3.b6"></a>The LRS MAY identify the user with any of the legal identifying properties if 
-a user connects directly (using HTTP Basic Authentication) or a part of 3-legged OAuth. 
 
-##### <a name="2.4.9.s4"></a>OAuth Credentials as Authority
+Note: To promote interoperability, LRSs are encouraged to try to map the principal that most logically "owns"
+the credential used to send statements to an authority when overwriting the authority as described above.
```

Comment:
See https://github.com/adlnet/xAPI-Spec/pull/1079/files#r228115287

I'd like to see this expanded.

Reply:
Or maybe remove it? Based on your other comment, it seems like you disagree with this guidance, though I do suggest that it is a good boiling down of the principle behind the auth-specific text that was removed.

Reply:
Whether or not I disagree with this depends on how you interpret ```most logically owns```. I don't think that's necessarily the user that creates the credential, but could be an account representing the system using the credentials. Perhaps we can say that.

Reply:
see: https://github.com/bscSCORM/xAPI-Spec/commit/83060cde45ff41cfeeb94564dddf2f85644a70c6